video/out/cocoa_common.m

```diff
@@ -574,7 +574,6 @@ int vo_cocoa_config_window(struct vo *vo)
         if (!s->embedded && s->window) {
             if (reset_size)
                 queue_new_video_size(vo, width, height);
-            vo_cocoa_fullscreen(vo);
```

Comment:
That function is completely private to cocoa, so I don't understand the question.

Just make sure ``--fs`` and switching FS at runtime with ``f`` works.